recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ source:
     - support-grpc-1.34.patch
 
 build:
-  number: 25
+  number: 26
   # for cuda on linux, building with 9.2 is enough to be compatible with all later versions,
   # since arrow is only using libcuda, and not libcudart.
   skip: true  # [linux and cuda_compiler_version not in ("None", "9.2")]
```

Comment:
@jaimergp As I see you updating the CUDA matrix, can you give me a hint what are the minimal CUDA versions we have on the OSes? I struggle with the migrator currently to understand what is added and what is removed.

Reply:
By default, we support 10.2, 11.0, 11.1, 11.2.

We have the infrastructure for older versions too. Linux starts on 9.2. Windows starts on 10.0. You have to opt-in in that case, and switch to "manual command". [This file](https://github.com/conda-forge/nvcc-feedstock/blob/master/recipe/conda_build_config.yaml) sums everything we have so far.

I am adding docs for this [here](https://github.com/conda-forge/conda-forge.github.io/pull/1262), so please let me know if there's something else that would need clarification. Maybe a table with the versions/OSes?

Reply:
For me the [PR](https://github.com/conda-forge/conda-forge.github.io/pull/1262) is sufficient, not sure whether more is needed for others. `arrow-cpp` is quite basic in its CUDA needs.

Reply:
If you don't do anything else, you'll be under the default behavior; i.e. building for 10.2 and above.

Reply:
Ah, but you only build for one version, I see. Then that last commit should be enough IMO.